application/dash_application/views/call_ready_rna.py

```diff
@@ -135,13 +135,15 @@ def get_merged_rna_data():
 
 
 def generate_five_to_three(df, graph_params):
-    return generate(
+    fig = generate(
        "5 to 3 Prime Bias", df,
        lambda d: d[util.ml_col],
        lambda d: d[RNASEQQC2_COL.MetricsMedian5PrimeTo3PrimeBias],
        "", graph_params["colour_by"], graph_params["shape_by"],
        graph_params["shownames_val"], [],
        util.ml_col)
+    fig.update_layout(yaxis_type="log")
```

Comment:
Would it make sense to also update the title on line 142 to be "Log Ratio"?

Reply:
Fixed. The strange y-axis numbering is a design choice by plotly: https://plot.ly/python/axes/#logarithmic-axes.

Reply:
👍 to updated axis title, 👎 to this design choice (given our particular use case scenario, where our data points are around 1).